iot/api-client/manager/manager_test.py

```diff
@@ -33,7 +32,7 @@
 es_cert_path = 'resources/ec_public.pem'
 rsa_cert_path = 'resources/rsa_cert.pem'
 rsa_private_path = 'resources/rsa_private.pem'  # Must match rsa_cert
-topic_id = 'test-device-events-{}'.format(int(time.time()))
+topic_id = 'test-device-events-{}'.format(uuid.uuid1())
```

Comment:
is uuid1 sufficiently random? uuid1 is a hash of machine address and time. I think it also has another identifier, but it may collide. I think we use uuid4 in a lot of the other tests for this reason.

Reply:
I thought the original author was thinking about the case where they need to manually delete the leaked resources, so I thought we should use time based uuid1, but whatever, I can use uuid4. Another commit is on the way.